x/staking/types/keys.go

```diff
@@ -264,23 +264,6 @@ func GetREDByValSrcIndexKey(delAddr sdk.AccAddress, valSrcAddr, valDstAddr sdk.V
 	return key
 }
 
-// GetREDByValDstIndexKey creates the index-key for a redelegation, stored by destination-validator-index
-// VALUE: none (key rearrangement used)
-func GetREDByValDstIndexKey(delAddr sdk.AccAddress, valSrcAddr, valDstAddr sdk.ValAddress) []byte {
-	REDSToValsDstKey := GetREDsToValDstIndexKey(valDstAddr)
-	offset := len(REDSToValsDstKey)
-
-	// key is of the form REDSToValsDstKey || delAddrLen (1 byte) || delAddr || valSrcAddrLen (1 byte) || valSrcAddr
-	key := make([]byte, offset+2+len(delAddr)+len(valSrcAddr))
-	copy(key[0:offset], REDSToValsDstKey)
-	key[offset] = byte(len(delAddr))
-	copy(key[offset+1:offset+1+len(delAddr)], delAddr.Bytes())
-	key[offset+1+len(delAddr)] = byte(len(valSrcAddr))
-	copy(key[offset+2+len(delAddr):], valSrcAddr.Bytes())
-
-	return key
-}
-
 // GetREDKeyFromValSrcIndexKey rearranges the ValSrcIndexKey to get the REDKey
 func GetREDKeyFromValSrcIndexKey(indexKey []byte) []byte {
```

Comment:
why is this needed?

Reply:
This is removed in other branch of migrating `REDKeyFromValSrcIndexKey` to collections (here: https://github.com/cosmos/cosmos-sdk/pull/17332)